ManiVault/src/private/HelpManager.cpp

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later 
+// A corresponding LICENSE file is located in the root directory of this source tree 
+// Copyright (C) 2023 BioVault (Biomedical Visual Analytics Unit LUMC - TU Delft) 
+
+#include "HelpManager.h"
+
+#include <Application.h>
+
+#include <QDesktopServices>
+
+using namespace mv::gui;
+using namespace mv::util;
+
+#ifdef _DEBUG
+    //#define HELP_MANAGER_VERBOSE
+#endif
+
+namespace mv
+{
+
+HelpManager::HelpManager() :
+    AbstractHelpManager(),
+    _showLearningCenterAction(this, "Learning center"),
+    _toDiscordAction(this, "Join us..."),
+    _toWebsiteAction(this, "Website"),
+    _toWikiAction(this, "Wiki"),
+    _toRepositoryAction(this, "Repository")
+{
+    _showLearningCenterAction.setIconByName("chalkboard-teacher");
+    _showLearningCenterAction.setToolTip("Go to the learning center");
+
+    _toWebsiteAction.setIconByName("globe");
+    _toWebsiteAction.setDefaultWidgetFlags(TriggerAction::Icon);
+
+    _toWikiAction.setIconByName("book");
+    _toWikiAction.setDefaultWidgetFlags(TriggerAction::Icon);
+
+    _toRepositoryAction.setIconByName("code-branch");
```

Comment:
Similar to above, we could use the [fontawesome github icon](https://fontawesome.com/icons/github) directly here.